charactersheet.html

```diff
@@ -0,0 +1,579 @@
+                                            <span>Deception</span>
+                                        </label>
+                                    </p>
+                                    <p>
+                                        <label>
+                                            <input type="checkbox" class="filled-in" id="intimidation" unchecked="checked" />
+                                            <span>Intimidation</span>
+                                        </label>
+                                    </p>
+                                    <p>
+                                        <label>
+                                            <input type="checkbox" class="filled-in" id="performance" unchecked="checked" />
+                                            <span>Performance</span>
+                                        </label>
+                                    </p>
+                                    <p>
+                                        <label>
+                                            <input type="checkbox" class="filled-in" id="persuasion" unchecked="checked" />
+                                            <span>Persuasion</span>
+                                        </label>
+                                    </p>
+                                </form>
+                            </div>
+                        </div>
+
+                        <!-- Spell Dropdown -->
+                        <div>
+                            <div id="addModel" class="modal black-text"></div>
+                            <a class='dropdown-button btn dropdownBtn spellbtn' style="width:50%" href='#'
+                                data-activates='spellDrop'>spells<i class="material-icons right">arrow_drop_down</i></a>
+                            <ul id='spellDrop' class='dropdown-content'>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!--Inventory Text Area-->
+            <div class="row">
+                <div class="col s6">
+                    <div class="card inv-card" style="width: 100%;">
+                        <div class="card-content white-text">
+                            <span class="card-title center-align">Inventory</span>
+                            <hr>
+                            <div class="inventory" id="inventory">
+                                <div class="row">
+                                    <div class="col-25">
+                                        <label for="inventory">My Bag of Holding</label>
+                                    </div>
+                                    <div class="col-75">
+                                        <textarea id="subject" name="subject" class="white-text" placeholder="What are you carrying?"
+                                            style="height:200px"></textarea>
```

Comment:
try to avoid inline styles, even something like `height: 200px` should have been something defined in a class